test/clients/azure/client.go

```diff
@@ -19,12 +35,41 @@ func NewClientFromEnvironment() (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	subscriptionID := os.Getenv("AZURE_SUBSCRIPTION_ID")
 
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
+	log := logrus.NewEntry(logrus.StandardLogger())
+
+	conf, err := realfakerp.NewConfig(log, true)
+	if err != nil {
+		return nil, err
+	}
+
+	var rpURL string
+	focus := []byte(config.GinkgoConfig.FocusString)
+	switch {
+	case adminRpFocus.Match(focus), fakeRpFocus.Match(focus):
```

Comment:
the reason this is succeeding without the Makefile being updated is that this is matching the Admin in "[CustomerAdmin]" - I'd have expected you would be looking for "[someTag]" in square brackets and updating the Makefile?

Reply:
@charlesakalugwu please raise an issue about this

Reply:
#890